configure

```diff
@@ -522,6 +522,12 @@ while true; do
   CUDA_TOOLKIT_PATH=""
 done
 
+# Set default CUDA version if not set
+if [ -z "$TF_CUDA_VERSION" ]; then
+  TF_CUDA_VERSION="8.0"
+  export TF_CUDA_VERSION
```

Comment:
I think I know what is going on.
Simply exporting it wont help. You need the value written to bazelrc, with this command:
`write_action_env_to_bazelrc "TF_CUDA_VERSION" "$TF_CUDA_VERSION"`
However, you need to make sure that we do not have two conflicting values are not written, so it may complicate things a little.